src/read.fs

```diff
@@ -610,6 +610,7 @@ let readAliasDeclaration (checker: TypeChecker) (d: TypeAliasDeclaration): FsTyp
     let name = d.name.getText()
     let asAlias() =
         {
+            Comments = readCommentsAtLocation checker d.name
```

Comment:
This doesn't seem to work, it doesn't get the comment in test/fragments/custom/delegate.d.ts. Any ideas?

Reply:
It does work in my local copy:
```fsharp
/// This does something
type Delegate1 =
    delegate of arg1: float * ?optArg1: string * ?optArg2: string -> unit
```

But this uncovered an issue with testing: comments aren't checked to match in both files. `Delegate1` in `*.expected.fs` doesn't contain the comment, while the generated `*.fs` file does. And yet the test passes.  
(which makes sense for the header comment (`// ts2fable 0.0.0`), but not for xml comments)


<br/>
<br/>

When your using the web-app, check if your really using `localhost:8080` and NOT `https://fable.io/ts2fable/`.  
I'm definitely not writing this because that just happened to me....

Reply:
Ah, you're right thanks! Actually I was just looking at the tests and wondered why it wasn't failing without the comment 😅